safekeeper/src/bin/safekeeper.rs

```diff
@@ -252,7 +260,8 @@ fn start_safekeeper(mut conf: SafeKeeperConf, given_id: Option<ZNodeId>, init: b
     let signals = signals::install_shutdown_handlers()?;
     let mut threads = vec![];
     let (callmemaybe_tx, callmemaybe_rx) = mpsc::unbounded_channel();
-    GlobalTimelines::set_callmemaybe_tx(callmemaybe_tx);
+    let (wal_backup_launcher_tx, wal_backup_launcher_rx) = mpsc::channel(100);
```

Comment:
> 100

Is that enough for our plans to have ~1000 tenants served by a pageserver?
I would assume, one safekeeper has to know about ~333 tenants at least which might start randomly blocking others during intensive writes?

Reply:
Every safekeeper will push here 1000 times. But that shouldn't take long, launcher just spins/kills the task and ready take the next one from the channel.